content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -104,46 +102,70 @@ generator client {
 
 Then you can pass an async function into [`$transaction`](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api).
 
-The following is an example of purchasing a movie ticket online. In this example, we really don't want to sell the same ticket twice:
+Let's say we're building an online banking system. One of the actions we want to perform is to send money from one person to another person.
+
+As experienced developers, we want to make sure that during the transfer,
+
+- the amount doesn't disappear
+- the amount isn't doubled
+
+This is a great use-case for interactive transactions because we need to perform logic in-between the writes to check the balance.
```

Comment:
```suggestion
This is a great use-case for interactive transactions because of the need to perform logic in-between the writes to check the balance.
```
I wonder about "the writes" since we don't mention writes before that. It might be a bit confusing.

Reply:
"Writes" are talked of on that page though (https://www.prisma.io/docs/concepts/components/prisma-client/transactions#nested-writes) and the concept of transactions is spoken about like "writing to the database" (https://www.prisma.io/docs/guides/performance-and-optimization/prisma-client-transactions-guide#dependent-writes)

Reply:
Changed it to "database writes"